Ports/halflife/package.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port="half-life"
+version="1.0.0"
+useconfigure="true"
+depends=("SDL2")
+workdir="xash3d-fwgs-master"
+files="https://github.com/SerenityPorts/xash3d-fwgs/archive/master.tar.gz xash3d_engine.tar.gz
+https://github.com/SerenityPorts/hlsdk-xash3d/archive/master.tar.gz xash3d_hldll.tar.gz"
+launcher_name="Half-Life"
+launcher_category="Games"
+launcher_command="sh /home/anon/Games/halflife/hl.sh"
```

Comment:
Could we maybe use `LD_LIBRARY_PATH=/home/anon/Games/halflife /home/anon/Games/halflife/xash3d` here? Or if that doesn't work, `sh -c 'LD_LIBRARY_PATH=/home/anon/Games/halflife /home/anon/Games/halflife/xash3d'`? That would allow us to skip the seperate launcher script entirely.

Reply:
Neither of those seem to work, I think the script is the best course of action for now. I'll update the generation to be the here doc style you've outlined above.